### PR TITLE
Market Card Adjustments

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/4093

adjusted market cards to be a bit tighter/cleaner as requested

### CODE REVIEW

File: packages/augur-ui/src/modules/market-cards/market-card.tsx [hunk 1]
```diff
@@ -177,140 +201,125 @@ export default class MarketCard extends React.Component<
...
+                canDispute={canDispute}
+                canSupport={canSupport}
+              />
+              {outcomesFormatted &&
+                outcomesFormatted.length > showOutcomeNumber &&
+                !expandedView && (
                   <button onClick={this.expand}>
                     <ChevronFlip
-                      stroke='#fff'
+                      stroke="#fff"
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.tsx]: formatter is off, we are using single quotes

Comment (JohnDanz) [reply packages/augur-ui/src/modules/market-cards/market-card.tsx]: ah, used prettier, will adjust

File: packages/augur-ui/src/modules/market-cards/market-card.tsx [hunk 2]
```diff
@@ -103,67 +120,74 @@ export default class MarketCard extends React.Component<
...
-            hoverText='Position'
+            hoverText="Position"
           />
-        }
-        {hasStaked &&
+        )}
+        {hasStaked && (
           <HoverIcon
-            label='dispute'
+            label="dispute"
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.tsx]: formatter is off, we are using sigle quotes

File: packages/augur-ui/src/modules/market-cards/market-card.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/market-cards/market-card.tsx]: we all are using prettier. the config file has use single quotes

Comment (bthaile) [reply packages/augur-ui/src/modules/market-cards/market-card.tsx]: in prettier.config
```
module.exports = {
  singleQuote: true,
  trailingComma: 'es5',
};

```

**Comment by bthaile** [comment]
tablet view category labels are overlaping

image

**Comment by bthaile** [comment]
reporting start time is wrapping which is correct. The location of the label isn't. needs to be at bottom right.
image

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🚀

 This link will always point to this PR's latest changes: https://peek.run/4kghv5l

**Comment by bthaile** [comment]
multiple instances of reporting ends lables

Screen Shot 2019-10-08 at 12 44 22 PM

**Review by bthaile — APPROVED** [review]
looks great!

**Pull request merged**